website/docs/examples/kubernetes/additionalPrinterColumns.md

```diff
@@ -0,0 +1,80 @@
+---
+sidebar_position: 20
+---
+
+# Additional Printer Columns
```

Comment:
Could you please inline this in the existing simpleschema doc. Instead of creating a new doc ?